vizro-ai/snyk/requirements.txt

```
@@ -1,6 +1,7 @@
 pandas
 tabulate
 openai>=1.0.0
+tenacity==8.4.1
```

Seems like the latest version seems to be working 🥳  Do you want to apply the suggested change from @antonymilne? Should be good to go then 👍 

```suggestion
tenacity!=8.4.1
```